### ISSUE TITLE
Disable ANSI codes in help output when running on Windows

### ISSUE THREAD

**Issue by pstemari** [GoogleCloudPlatform/gsutil#179]
Currently ANSI escape codes are used to provide highlighting, etc, in help output.  This should be turned off when running under Windows, since console windows simply print the codes along with the text.

For example:

```
←[1mSET←[0;0m
  The "gsutil web set" command will allow you to set Website Configuration
  on your bucket(s). The "set" sub-command has the following options:
```

**Comment by hkdennis2k** [comment]
It is not a true statement.
Windows 10 console do support color.

Git-for-windows project has managed to color output under both cmd/PS/msys by default

**Comment by pstemari** [comment]
You can color output, but not by just sending out escape sequences.

You have to call Windows-specific APIs (SetConsoleMode with ENABLE_VIRTUAL_TERMINAL_PROCESSING) in order to do it. Currently those are not called by the python driver.

https://msdn.microsoft.com/en-us/library/windows/desktop/mt638032(v=vs.85).aspx
https://msdn.microsoft.com/en-us/library/windows/desktop/ms686033(v=vs.85).aspx

**Comment by thobrla** [comment]
This was fixed back in 4.16 with https://github.com/GoogleCloudPlatform/gsutil/commit/6005cae1592c8436bd24a47d4ce6863b77cf351c#diff-f340596acc95ed802e9e7c7b48b0cde8
